resources/qml/SidebarSimple.qml

```diff
@@ -270,46 +254,61 @@ Item
         }
 
         Label{
-            id: supportHelperLabel
+            id: supportExtruderLabel
+            visible: (supportEnabled.properties.value == "True") && (machineExtruderCount.properties.value > 1)
             anchors.left: parent.left
             anchors.leftMargin: UM.Theme.getSize("default_margin").width
-            anchors.verticalCenter: supportCheckBox.verticalCenter
+            anchors.verticalCenter: supportExtruderCombobox.verticalCenter
             width: parent.width / 100 * 35 - 3 * UM.Theme.getSize("default_margin").width
-            //: Support label
-            text: "";
+            text: catalog.i18nc("@label", "Support Extruder");
             font: UM.Theme.getFont("default");
             color: UM.Theme.getColor("text");
         }
 
-        CheckBox{
-            id: supportCheckBox
-            visible: machineExtruderCount.properties.value <= 1
-            property alias _hovered: supportMouseArea.containsMouse
+        ComboBox {
+            id: supportExtruderCombobox
+            visible: (supportEnabled.properties.value == "True") && (machineExtruderCount.properties.value > 1)
+            model: extruderModel
 
-            anchors.top: adhesionCheckBox.bottom
-            anchors.topMargin: UM.Theme.getSize("default_margin").height
-            anchors.left: supportHelperLabel.right
+            anchors.top: enableSupportCheckBox.bottom
+            anchors.topMargin: {
+                if ((supportEnabled.properties.value == "True") && (machineExtruderCount.properties.value > 1)) {
+                    return UM.Theme.getSize("default_margin").height;
+                } else {
+                    return 0;
+                }
```

Comment:
Brackets on a new line. Fixed that for you in d448ed7e0863ec8c6d72f8569a586e4d594e2af5.